ocaml-lsp-server/docs/ocamllsp/documentation-spec.md

```diff
@@ -0,0 +1,40 @@
+# Documentation Request
+
+## Description
+
+Merlin has a command `document` that gets `odoc` documentation for symbols based on a cursor position or an optional identifier. This request allows documentation to be gotten using a classic Merlin workflow.
```

Comment:
I think it would be better to make the definition of `merlin` agnostic and explain concretely what the query does. (By explaining that it allows you to bypass `Hover`, for example, to bring up documentation information).